### DISCUSSION TITLE
Effort justification and automation in outbreak analytics pipelines

### DISCUSSION THREAD

**Discussion by jamesmbaazam** [epiverse-trace/epiverse-trace.github.io#121] [General]
I read about the [effort justification effect](https://en.wikipedia.org/wiki/Effort_justification) today and it got me thinking about automation versus manual tasks in outbreak analytics pipelines. Basically, effect justification is when people feel attached to an output due to the effort put into acquiring/achieving it. It's also referred to as the IKEA effect. Apparently, people are sometimes more attached to IKEA products because they get to build them themselves and I guess this extends to DIY in general. 

This same effect raises a lot of debate in many spheres of life, including, how one feels using `git` from the terminal versus GUI, `SPSS` versus `R`, driving a manual car vs automatic, etc. 

In outbreak analytics workflow/pipeline design and automation, to what extent can this be applied? For example, do we have any sense of what aspects of outbreak analytics pipelines and workflows users prefer to perform manually rather than through automation? How can we apply this effect in the design of our software?

**Comment by pratikunterwegs** [comment]
Thanks @jamesmbaazam - that's an interesting discussion. I'm not sure whether this particularly applies to software usage. Often, the main selling point of a package, GUI, or software is that it reduces a complex wofklow to "just one click".

**Comment by jamesmbaazam** [comment]
> Often, the main selling point of a package, GUI, or software is that it reduces a complex workflow to "just one click".

Or maybe this applies when the selling point is that it gives you more control over what you can do rather than having to deal with a black box 😉.
